### PR TITLE
Rpi serial cable added position 10 ( fixed #289 )

### LINKED ISSUE #289: add an other optional to the requirements
please add as 10th position
rpi serial cable https://www.amazon.com/s?k=raspberry+pi+serial+cable

### PR DESCRIPTION
**Pull request by jamjakpa**
Fixed #289 
### Description
RPI serial cable added on position 10 ( fixed#289 )

rpi_Serial_cable

### Raw.Githack preview link
raw.githack link to page(s) changed.

https://raw.githack.com/jamjakpa/jamjakpa.github.io/rpi_serial_cable/#!./pages/vi/firststeps.md

### CODE REVIEW

**Comment by pattanawadee88** [comment]
the raw.githack link is not working.

**Comment by jamjakpa** [comment]
> the raw.githack link is not working.

raw.githack link fixed

File: pages/vi/firststeps.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ The goal of OLE is to provide educational infrastructure to learners and educato
 	- [Raspberry Pi Camera]("https://www.raspberrypi.org/products/camera-module-v2/")
 	- [Raspberry Pi Night Vision Camera]("https://www.raspberrypi.org/products/pi-noir-camera-v2/")
 	- [generic Raspberry Pi 5 Megapixel Camera]("https://www.amazon.com/AuviPal-Megapixels-Raspberry-Camera-Module/dp/B07PQ63D2S/")
+	- [Raspberry Serial Cable]("https://www.amazon.com/s?k=raspberry+pi+serial+cable")
```
Comment (pattanawadee88) [inline pages/vi/firststeps.md]: The link is not working.

**Review by pattanawadee88 — CHANGES_REQUESTED** [review]
Delete jamjakpa.github.io this file.

File: pages/vi/firststeps.md (hunk 1 above)
Comment (CalebProvost) [inline pages/vi/firststeps.md]: Somewhere along the line these links has quotes placed around them.
.md doesn't redirect to the link when they're in quotes. Remove them and the problem should be resolved.

**Review by CalebProvost — CHANGES_REQUESTED** [review]

File: pages/vi/firststeps.md (hunk 1 above)
Comment (kjong) [reply pages/vi/firststeps.md]: Looks like the quotes are still there for the other links above.

**Comment by dogi** [comment]
@jamjakpa since this PR is also causing merge confict I am closing this PR without merging ...

please fix your forked master and try again

**Pull request closed without merging**